staff/daniel-toledo/precourse/calculadora/main.js

```diff
@@ -1,5 +1,4 @@
 
```

Comment:
WOW!!!! this file should not be committed in this branch! Try to fix it, deleting it from here and committing it from the precourse branch. 🤓